test/k8sT/Updates.go

```diff
@@ -114,6 +114,8 @@ func InstallAndValidateCiliumUpgrades(kubectl *helpers.Kubectl, oldVersion, newV
 			"cilium-cm-patch-clean-cilium-state.yaml",
 		)
 		ExpectWithOffset(1, err).To(BeNil(), "Cilium %q was not able to be deployed", newVersion)
+
+		ExpectAllPodsTerminated(kubectl)
```

Comment:
should this function be instead `WaitAllPodsToBeTerminated`?

Reply:
No, if the pods did not terminate in the time-out the build should fail. It's the standard in all test.